components/automate-ui/src/app/pages/project/list/project-list.component.html

```diff
@@ -5,9 +5,12 @@
   <main>
     <chef-page-header>
       <chef-heading>Projects</chef-heading>
-      <chef-subheading>
+      <chef-subheading *ngIf="(sortedProjects$ | async)?.length < 6">
         Projects are named groups of resources used to define policies.
       </chef-subheading>
+      <chef-subheading *ngIf="(sortedProjects$ | async)?.length >= 6">
+        Projects group resources together. Max of 6 projects allowed.
```

Comment:
Please, please, please put that magic number used throughout into a named constant. Even on this line.